tools/auto/presenceValidator.ts

```diff
@@ -0,0 +1,280 @@
+import "source-map-support/register.js";
+
+import actions from "@actions/core";
+import { existsSync, readFileSync } from "fs";
+import { readFile, stat } from "fs/promises";
+import got from "got";
```

Comment:
- can we pick if we use axios, _got_ or fetch? like each repo uses something else
- why not just use fetch

Reply:
I prefer got (esm only) fetch is to plain